bot/handlers/whois.py

```diff
@@ -8,6 +8,45 @@
 from users.models.user import User
 
 
+@is_club_member
+def whos_joined(update: Update, context: CallbackContext) -> None:
+
+    original_message = update.message  # look at the author of this message (works only in private chats)
+    from_user = original_message.from_user
+    from_username = original_message.chat.username
+
+    context.bot.delete_message(chat_id=update.message.chat_id, message_id=update.message.message_id)
+
+    if from_user.is_bot:
+        update.message.reply_text(
+            f"🤖В ч@т з@ш3л 60т: {from_user.id}",
+            quote=True
+        )
+        return None
+
+    telegram_id = from_user.id
+    user = User.objects.filter(telegram_id=telegram_id).first()
+
+    whos_joined = from_username if from_username else telegram_id
+    if not user:
+        update.message.reply_text(
+            f"🤨 К чату присоединился {whos_joined}, которого нет в Клубе. Гоните его, насмехайтесь над ним!",
```

Comment:
Смущает, что тут может написаться "К чату присоединился 1234567...". Наверное, лучше брать имя профиля.

Reply:
Может стоит делать как меншн?
```suggestion
            f"🤨 К чату присоединился @{whos_joined}, которого нет в Клубе. Гоните его, насмехайтесь над ним!",
```